docs/src/inputs/prompt/examples/exampleWarningPrompt.jsx

```diff
@@ -0,0 +1,45 @@
+import {
+    Button,
+    Prompt,
+} from 'react-cm-ui';
+import React, { useState } from 'react';
+
+function ExampleWarningPrompt() {
+    const [showPrompt, setShowPrompt] = useState(false);
+
+    function onPromptClick() {
+        setShowPrompt(true);
+    }
+
+    function onYesClick() {
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
```

Comment:
This example needs all three states shown as in:

![image](https://user-images.githubusercontent.com/1100790/98309108-8012fb00-1f7e-11eb-8bb3-b0bfda512682.png)

Reply:
Fixed